projects/fal/src/fal/cli/doctor.py

```diff
@@ -0,0 +1,38 @@
+import os
+import platform
+
+
+def _doctor(args):
+    import isolate
+    from rich.table import Table
+
+    import fal
+
+    table = Table(show_header=False, show_lines=False, box=None)
+    table.add_column("name", no_wrap=True, style="bold")
+    table.add_column("value", no_wrap=True)
+
+    table.add_row("fal", fal.__version__)
+    table.add_row("isolate", isolate.__version__)
+
+    table.add_row("", "")
+    table.add_row("python", platform.python_version())
+    table.add_row("platform", platform.platform())
+
+    table.add_row("", "")
+    table.add_row("FAL_HOST", fal.flags.GRPC_HOST)
+    FAL_KEY = os.getenv("FAL_KEY", "")
+    table.add_row("FAL_KEY", (FAL_KEY[:9] + "...") if FAL_KEY else "")
```

Comment:
maybe show the entire left part of the `:` so it is easy to id it for us

Reply:
Forgot to push, so had to create a followup https://github.com/fal-ai/fal/pull/242